### PR TITLE
Add documentation for `publish_streams`

### PR DESCRIPTION
**Pull request by pjotor**
API v2 now supports creating and updating Adverts with
`publication_streams`. This is now reflected in the API documentation.

Marketplacer repo PR: https://github.com/marketplacer/marketplacer/pull/12018

### CODE REVIEW

File: content/v2/adverts.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Name | Type | Description
 `map_brand`|`string` or `array of strings`| The value/s to be mapped to the brand.
 `code`|`string`| Optional - not shown to shoppers.
 `notes`|`string`| Optional - not shown to shoppers.
+`publish_streams`|`array of strings`| Optional - advert p[ublication streams for Marketplace and/or M Store, not shown to shoppers.
```
Comment (robotdana) [inline content/v2/adverts.md]: typo p[ublication

File: content/v2/adverts.md [hunk 2]
```diff
@@ -145,6 +146,7 @@ or
       map_brand: ["Bikes - Hybrid Bikes", "Bikes - Giant Bikes"],
       map_taxon: ["Bikes - Hybrid Bikes", "Bikes - Giant Bikes"],
       published: false,
+      publishe_streams: ["m_store"],
```
Comment (robotdana) [inline content/v2/adverts.md]: another typo

File: content/v2/adverts.md (hunk 1 above)
Comment (robotdana) [reply content/v2/adverts.md]: perhaps list the literal strings we expect (marketplace, m_store)

Comment (pjotor) [reply content/v2/adverts.md]: Sausage fingers x 2

Comment (pjotor) [reply content/v2/adverts.md]: I thought about that, but decided against it because different resellers and stores will have different streams available.

Comment (robotdana) [reply content/v2/adverts.md]: do we tell them what they could be on the error message then?

File: content/v2/adverts.md [hunk 3]
```diff
@@ -95,6 +95,7 @@ Name | Type | Description
 `map_brand`|`string` or `array of strings`| The value/s to be mapped to the brand.
 `code`|`string`| Optional - not shown to shoppers.
 `notes`|`string`| Optional - not shown to shoppers.
+`publish_streams`|`array of strings`| Optional - advert publication streams for Marketplace and/or M Store, not shown to shoppers.
```
Comment (robotdana) [inline content/v2/adverts.md]: even: Optional - advert publication streams for `marketplace` and/or `m_store`, not shown to shoppers.

File: content/v2/adverts.md (hunk 1 above)
Comment (pjotor) [reply content/v2/adverts.md]: No, the API error handling does not have that level of transparency - even if the model have.

**Pull request merged**